easybuild/easyconfigs/l/Lucene-Geo-Gazetteer/Lucene-Geo-Gazetteer-20170718.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PackedBinary'
+
+name = 'Lucene-Geo-Gazetteer'
+version = '20170718' 
+
+homepage = 'https://github.com/chrismattmann/lucene-geo-gazetteer'
+description = """A command line gazetteer built around the Geonames.org dataset, that uses the Apache Lucene library
+ to create a searchable gazetter."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/chrismattmann/lucene-geo-gazetteer/archive/']
+sources = [{'filename': SOURCE_TAR_GZ, 'download_filename': '6c38272.tar.gz'}]
+checksums = ['2ed355bfdcf865d815df9dc52ee747eeb8a15d065ef159ebf933ad36e1555ad4']
+
+builddependencies = [('Maven', '3.5.0')]
+dependencies = [('Java', '')]
```

Comment:
@boegel This is odd - does it really only need *any* version of `Java`?

Reply:
Hah! This is obviously wrong, it should specify a particular version of Java, will fix.